apps/passport-server/migrations/57_password_reset_limit.sql

```diff
@@ -0,0 +1,94 @@
+CREATE TABLE rate_limit_types(
+  event_type VARCHAR NOT NULL,
+  hourly_limit INT NOT NULL,
+  PRIMARY KEY(event_type)
+);
+
+CREATE TABLE rate_limit_buckets(
+  event_type VARCHAR NOT NULL,
+  event_id VARCHAR NOT NULL,
+  remaining INT NOT NULL,
+  last_refill TIMESTAMPTZ,
+  PRIMARY KEY(event_type, event_id),
+  FOREIGN KEY(event_type) REFERENCES rate_limit_types
+);
+
+-- Allow 10 password reset attempts per hour
```

Comment:
Does this really apply only to password resets?  We also send email tokens for normal login.  It's probably appropriate to limit those too, so maybe it's just the comment which needs an update.